src/ai/backend/manager/services/vfolder/exceptions.py

```diff
@@ -25,6 +25,19 @@ def error_code(cls) -> ErrorCode:
         )
 
 
+class VFolderInvitationNotFound(BackendAIError, web.HTTPNotFound):
+    error_type = "https://api.backend.ai/probs/vfolder-invitation-not-found"
+    error_title = "Virtual folder invitation not found."
+
+    @classmethod
+    def error_code(cls) -> ErrorCode:
+        return ErrorCode(
+            domain=ErrorDomain.VFOLDER,
+            operation=ErrorOperation.READ,
```

Comment:
I think it would be too fine-grained to have a separate VFOLDER_INVITATION domain just for this one case.

Reply:
Please add this.

Reply:
Ok, I will add it though the original comment was intended to suggest not adding new domain..